python/ray/memory_monitor.py

```diff
@@ -54,7 +54,10 @@ def __init__(self, error_threshold=0.95, check_interval=1):
         self.check_interval = check_interval
         self.last_checked = time.time()
         self.error_threshold = error_threshold
-        if not psutil:
+        # TODO: This is a hotfix and should be removed later. We check for
+        # handlers because this object is instantiated before
+        # logging is configured which leads to non-descript messages printed.
+        if not psutil and logger.handlers:
```

Comment:
This gets rid of `No handler found for logger 'ray.memory_monitor'`

Reply:
Please fix this.